Cakefile

```diff
@@ -445,6 +447,8 @@ runTests = (CoffeeScript) ->
         passedTests++
     catch err
       onFail description, fn, err
+  global.skip = (description, fn) ->
+    doSkip description
```

Comment:
Rather than introducing a general “skip test” helper, perhaps we can narrow down when the failing tests fail (like particular Node major versions, say) and add `return if` lines similar to `return if global.testingBrowser` to only disable them in the known problematic environments.